ontotext-yasgui-web-component/src/models/yasgui-configuration.ts

```diff
@@ -71,3 +71,29 @@ export enum Orientation {
   VERTICAL = 'orientation-vertical',
   HORIZONTAL = 'orientation-horizontal'
 }
+
+export const defaultOntotextYasguiConfig: Record<string, any> = {
+  render: RenderingMode.YASGUI,
+  orientation: Orientation.VERTICAL,
+  showEditorTabs: true,
+  showResultTabs: true,
+  showToolbar: false
+}
+
+export const defaultYasguiConfig: Record<string, any> = {
+  copyEndpointOnNewTab: true,
+  endpoint: '',
+  method: 'POST',
+  headers: () => {
+    return {
+      'Content-Type': 'application/x-www-form-urlencoded',
+      'Accept': 'application/sparql-results+json',
+      'X-GraphDB-Local-Consistency': 'updating'
+    };
+  }
+}
+
+export const defaultYasqeConfig: Record<string, any> = {
+  query: '',
```

Comment:
IMO would be good to set default ontotext default query `select * where { 
	?s ?p ?o .
} limit 100 `